css/templates/travel/elements/_overlay.css

```diff
@@ -0,0 +1,68 @@
+
+/**
+ * overlay
+ *
+ * Create a full-page overlay. This is similar to an amp-lightbox with the
+ * important difference that it can be wholly controlled with css and disabled
+ * at different breakpoints.
+ *
+ * The required markup is:
+ *  div.overlay(.overlay-visible) > div.overlay-overflow > div.overlay-content
+ *
+ * Modifier Classes:
+ * - overlay-visible
+ */
+
+.travel-overlay {
```

Comment:
I think a lot of the properties can be moved to html?

Reply:
done